.github/workflows/test.yml

```diff
@@ -54,9 +54,12 @@ jobs:
       - name: Pull Docker image
         run: docker-compose pull
 
-      - name: Run linters
+      - name: Run code linters
         run: docker-compose run test yarn lint
 
+      - name: Run API docs linters
```

Comment:
It would be best to have a separate job for it so that 1) it can be run in parallel or 2) deleted when not used

Reply:
Improved in https://github.com/nimblehq/nimble-survey-web/pull/91/commits/da404c8083ee058c263a578b49f2d934d3107c6b